integrations/fastembed/src/haystack_integrations/components/rankers/fastembed/ranker.py

```diff
@@ -0,0 +1,194 @@
+from typing import Any, Dict, List, Optional
+
+from haystack import Document, component, default_from_dict, default_to_dict, logging
+
+from .ranker_backend.fastembed_backend import _FastembedRankerBackendFactory
+
+logger = logging.getLogger(__name__)
+
+
+@component
+class FastembedRanker:
+    """
+    Ranks Documents based on their similarity to the query using [Fastembed models](https://qdrant.github.io/fastembed/examples/Supported_Models/).
+
+    Documents are indexed from most to least semantically relevant to the query.
+
+    Usage example:
+    ```python
+    from haystack import Document
+    from haystack_integrations.components.rankers.fastembed import FastembedRanker
+
+    ranker = FastembedRanker(model_name="Xenova/ms-marco-MiniLM-L-6-v2", top_k=2)
+
+    docs = [Document(content="Paris"), Document(content="Berlin")]
+    query = "What is the capital of germany?"
+    output = ranker.run(query=query, documents=docs)
+    docs = output["documents"]
+    ```
+    """
+
+    def __init__(
+        self,
+        model_name: str = "Xenova/ms-marco-MiniLM-L-6-v2",
+        top_k: int = 10,
+        cache_dir: Optional[str] = None,
+        threads: Optional[int] = None,
+        batch_size: int = 64,
+        parallel: Optional[int] = None,
+        local_files_only: bool = False,
+        meta_fields_to_embed: Optional[List[str]] = None,
+        meta_data_separator: str = "\n",
+    ):
+        """
+        Creates an instance of the 'FastembedRanker'.
+
+        :param model_name: Fastembed model name. Check the list of supported models in the [Fastembed documentation](https://docs.Fastembed.com/docs/models).
```

Comment:
wrong URL

Reply:
Indeed. I changed it
Note: the models are not there yet. I made a PR for this on the fastembed part (https://github.com/qdrant/fastembed/pull/393). It's merged, but I think it needs a release to be pushed on the website